src/common/btree.c

```diff
@@ -379,6 +379,67 @@ do {									\
 		(trace)->tr_at,	## __VA_ARGS__);			\
 } while (0)
 
+void
+hkey_common_gen(d_iov_t *key_iov, void *hkey)
```

Comment:
minor...this doesn't really belong in generic btree code.  Not sure where else to put it but this seems weird.